app/main.py

```diff
@@ -1,2 +1,65 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.length = 0
+        self.load = int(self.capacity * 2 / 3)
+        self.tab = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+        # preparing index of tab
+        hashed_key = hash(key)
+        index = hashed_key % self.capacity
+        while True:
+            # if new key length increases
+            if not len(self.tab[index]):
```

Comment:
```suggestion
            if self.tab[index] == []:
```
It's better to check explicitly

Reply:
it was my first variant)

Reply:
pycharm wants "is"